handler/population_types.go

```diff
@@ -46,11 +49,13 @@ func (h *PopulationTypes) Get(w http.ResponseWriter, req *http.Request) {
 	log.Info(ctx, "population types found", logData)
 	lpt := len(ptypes)
 
+	paginated, limitInt, offsetInt := manualPagination(limit, offset, ptypes)
```

Comment:
If these values are only used in the case of publicEndpoints being enabled this should probably be within the if scope below